android/build.gradle

```diff
@@ -37,6 +37,6 @@ repositories {
 
 dependencies {
   //noinspection GradleDynamicVersion
-  api "com.android.support:appcompat-v7:${rootProject.ext.supportLibVersion}"
+  api "androidx.appcompat:appcompat:${rootProject.ext.supportLibVersion}"
```

Comment:
you can remove this because AndroidX already bundled in RN

Reply:
@dulmandakh isn't AndroidX only for RN 0.60+ though? Making removing this backwards incompatible for older RN versions?

Reply:
IIRC, **appcompat** from Android Support Library is bundled in React Native since 0.59. So 0.60 will included from AndroidX too. Thus, it's not required to have appcompat as dependency as it'll add extra work.

Reply:
it's included in 0.58.

Reply:
I see 'javax.inject:javax.inject:1' and 'com.android.support:appcompat-v7:28.0.0' bundled into RN 58.4, but not AndroidX. I think we do have to include it here and in the example project?

Reply:
We also may want to just hold on this PR until 60 comes out?